src/main/java/com/thealgorithms/maths/Mean.java

```diff
@@ -0,0 +1,54 @@
+package com.thealgorithms.maths;
+
+/**
+ * https://en.wikipedia.org/wiki/Mean
+ * <p>
+ * by: Punit Patel
+ */
+public class Mean {
+
+    /**
+     * Arithmetic / Pythagorean Mean = (x₁ + x₂ + ... +  xₙ) / n
+     */
+    public static double arithmeticMean(double[] numbers) {
+        double sum = 0;
+        if (numbers.length == 0) {
+            return sum;
```

Comment:
Returning `0` might be very misleading in such situation. Please note that some of the tests need to be updated.
```suggestion
            return Double.NAN;
```

Reply:
Will do that.